client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -263,6 +268,18 @@ class Upload extends React.Component {
 		);
 	}
 
+	renderNotAvailable() {
+		return (
+			<EmptyContent
+				title={ this.props.translate( 'Sorry, themes upload is not yet available for multi-site installations.' ) }
```

Comment:
Copy suggestion:

"Uploading themes is not available for multi-site installations."

Generally we shouldn't apologize in these situations